src/cython/gudhi/alpha_complex.pyx

```diff
@@ -106,6 +110,8 @@ cdef class AlphaComplex:
         :returns: A simplex tree created from the Delaunay Triangulation.
         :rtype: SimplexTree
         """
-        simplex_tree = SimplexTree()
-        self.thisptr.create_simplex_tree(simplex_tree.thisptr, max_alpha_square)
-        return simplex_tree
+        stree = SimplexTree()
+        cdef intptr_t stree_int_ptr=stree.thisptr
+        cdef Simplex_tree_interface_full_featured* stree_ptr = <Simplex_tree_interface_full_featured*>stree_int_ptr
```

Comment:
Does cython complain if you try to save a line or two? Cast `stree.thisptr` without creating a variable `stree_int_ptr` for instance? It is distracting to see many lines of code that essentially do nothing (unless cython does require them).

Reply:
I tried this but cython complains.

```
        stree = SimplexTree()
        cdef Simplex_tree_interface_full_featured* stree_ptr = <Simplex_tree_interface_full_featured*>(stree.thisptr)
                                                              ^
------------------------------------------------------------

alpha_complex.pyx:115:63: Python objects cannot be cast to pointers of primitive types
```